x/tokenfactory/simulation/sim_msgs.go

```diff
@@ -0,0 +1,36 @@
+package simulation
+
+import (
+	"math/rand"
+
+	simulation "github.com/osmosis-labs/osmosis/v7/simulation/types"
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/keeper"
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// RandomMsgCreateDenom creates a random tokenfactory denom that is no greater than 44 alphanumeric characters
+func RandomMsgCreateDenom(k keeper.Keeper, sim *simulation.SimCtx, ctx sdk.Context) (*types.MsgCreateDenom, error) {
+	// select a pseudo-random simulation account
+	sender, err := sim.RandomSimAccountWithBalance(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// select a pseudo-random denom using alpha-numeric characters
+	var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890")
+	// denom can be no greater than 44 characters
+	denomLen := simulation.RandLTBound(sim, 44)
+
+	b := make([]rune, denomLen)
+	for i := range b {
+		b[i] = letterRunes[rand.Intn(len(letterRunes))]
+	}
+	subdenom := string(b)
```

Comment:
```suggestion
	subdenom := sim.RandStringOfLength(44)
```

Reply:
A+ for creativity though right

Reply:
F for execution

Reply:
haha!

For sure! The way this RandStringOfLength function got made in the SDK was basically me doing the exact same thing you did lolll